to-do-app/src/components/Input.js

```diff
@@ -0,0 +1,54 @@
+import React, { useReducer } from "react";
+import { useState } from "react";
+
+const Input = (props) => {
+    const [content, setContent] = useState('')
+
+    const handleChange = (event) => {
+        setContent(event.target.value)
+    }
+
+    const validateText = (text) => {
+        let isTextValid = false
+        if (text !== '') {
+            isTextValid = true
+        }
+        else {
+            isTextValid = false;
+        }
+
+        return isTextValid
+    }
+
+    const handleKeyDown = (event) => {
+        if (event.key === 'Enter') {
+            let isTextValid = validateText(event.target.value)
```

Comment:
No changes to `isTextValid`  use const instead of let

Reply:
you're right, thanks. fixed.